config/core/lab-configs.yaml

```diff
@@ -9,7 +9,9 @@ labs:
       - passlist:
           plan:
             - baseline
-            - kselftest
```

Comment:
The commit subject should mention lab-baylibre, otherwise it looks like this is about the test plan definition.  For example: `lab-configs.yaml: enable kselftest collections in lab-baylibre`